tsconfig.json

```diff
@@ -22,5 +22,5 @@
     "plugins": [{ "name": "typescript-plugin-css-modules" }]
   },
   "include": ["next-env.d.ts", "src/definitions.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "src/types/contracts"]
```

Comment:
The linter was complaining. These are generated by `typechain` so I would deem them type-safe.